shared/xccdf/services/base.xml

```diff
@@ -117,6 +117,7 @@ multiple processors.
 <ocil><systemd-check-macro enable="false" service="irqbalance" /></ocil>
 <rationale>In an environment with multiple processors (now common), the irqbalance service
 provides potential speedups for handling interrupt requests.</rationale>
+<platform idref="cpe:/o:virtualization:baremetal"/>
```

Comment:
@mpreisler Yes, generated datastrem is marked as platform RHEL7 at Benchmark level.